### PR TITLE
Feature/spl 8270 limit reattempts on healthchecks

### PR DESCRIPTION
**Pull request by lerics**
Fixing the hanging behaviour on healthchecks and (hopefully) spamming activeMQ on the way up. 
The latter is harder to verify.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -8,7 +8,7 @@
     </parent>
     <groupId>com.kjetland.dropwizard</groupId>
     <artifactId>dropwizard-activemq</artifactId>
-    <version>0.5.5-NOVAMEDIA</version>
+    <version>0.5.6-NOVAMEDIA-SNAPSHOT</version>
```
Comment (iulvalSPL) [inline pom.xml]: no SNAPSHOT in master (or maybe you want to merge this into develop first)

File: src/main/java/com/kjetland/dropwizard/activemq/ActiveMQBundle.java [hunk 2]
```diff
@@ -69,13 +63,26 @@ public void init(ActiveMQConfig activeMQConfig, Environment environment) {
...
+            activeMQConnectionFactory.setUserName(username);
+            activeMQConnectionFactory.setPassword(password);
+        }
+        return activeMQConnectionFactory;
+    }
+
     private void registerHealthCheckIfRequired(ActiveMQConfig activeMQConfig, Environment environment) {
-        if (activeMQConfig.healthcheckRequired) {
-            // Must use realConnectionFactory instead of (pooled) connectionFactory for the healthCheck
-            // Is needs its own connection since it is both sending and receiving.
```
Comment (iulvalSPL) [inline src/main/java/com/kjetland/dropwizard/activemq/ActiveMQBundle.java:75]: I think we should not lose this line in the comment that you've added at line 82

> // Is needs its own connection since it is both sending and receiving.

File: src/main/java/com/kjetland/dropwizard/activemq/ActiveMQConfig.java [hunk 3]
```diff
@@ -11,6 +11,9 @@
     @NotNull
     public String brokerUrl;
 
+    @JsonProperty
+    public String healthCheckAppendToBrokerUrl;
```
Comment (iulvalSPL) [inline src/main/java/com/kjetland/dropwizard/activemq/ActiveMQConfig.java:20]: Would be nice to add this parameter to the[ Readme](https://github.com/novamediasverige/dropwizard-activemq-bundle/blob/develop/README.md) file (same comment for healthCheckRequired, even though it was not part of this PR). 
Is there a way to set a default for this value?

**Review by iulvalSPL — APPROVED** [review]
👍 
Minor comments

File: pom.xml (hunk 1 above)
Comment (lerics) [reply pom.xml]: Yes, definitely develop first

File: src/main/java/com/kjetland/dropwizard/activemq/ActiveMQConfig.java (hunk 3 above)
Comment (lerics) [reply src/main/java/com/kjetland/dropwizard/activemq/ActiveMQConfig.java:20]: unfortunately no default value since it's dependent on whether other options are used or not in the brokerUrl. We couldn't find a way to set it in java without parsing brokerUrl either :/

Comment (lerics) [reply src/main/java/com/kjetland/dropwizard/activemq/ActiveMQConfig.java:20]: Fixed readme and some comments in code
